### DISCUSSION TITLE
School-issued email required but we didn't get one

### DISCUSSION THREAD

**Discussion by FrancocDev** [community/community#61306] [GitHub Education]
### Hi everyone,

Hi everyone, I'm trying to register to the education pack but after select my university, it keep me asking for an institutional email even when we didn't get one as a students on my faculty. 
Our University, on some faculties give students email, but mine didn't.
I can provide any documentation to verify that I'm studying on that University, but the form didn't allow me to verify without an email.
Thanks in advance and sorry for my english.

**Comment by nonvenomous** [comment]
Hi there, I'm in the same situation.
Here a bigger thread https://github.com/orgs/community/discussions/59360
